README.md

```diff
@@ -35,22 +35,24 @@ It uses [binary search](https://en.wikipedia.org/wiki/Binary_search_algorithm) t
-ruby            | [Oj](https://github.com/ohler55/oj)                         | ∞             | ∞             |
-Haskell         | [Aeson](https://hackage.haskell.org/package/aeson)          | ∞             | ∞             | available RAM is the only limit
+language        | json library                                                                     | nesting level | file size     | notes                         |
+----------------| -------------------------------------------------------------------------------- | ------------- | ------------- | ----------------------------- |
+C#              | [System.Text.Json](https://docs.microsoft.com/en-us/dotnet/api/system.text.json) | 64            | 128 bytes     | default MaxDepth is 64
+ruby            | [json](https://rubygems.org/gems/json/versions/1.8.3)                            | 101           | 202 bytes     |
+rust            | [serde_json](https://docs.serde.rs/serde_json/)                                  | 128           | 256 bytes     |
+php             | `json_decode`                                                                    | 512           | 1024 bytes    | maximum depth is configurable |
+perl            | [JSON::PP](https://perldoc.perl.org/JSON/PP.html)                                | 513           | 1026 bytes    |
+python3         | [json](https://docs.python.org/3/library/json.html)                              | 994           | 2.0 KB        | without sys.setrecursionlimit
+C               | [jansson](https://jansson.readthedocs.io/)                                       | 2049          | 4.0 KB        |
+java            | [Gson](https://github.com/google/gson)                                           | 5670          | 11.3 KB       |
+javascript      | `JSON.parse`                                                                     | 5713          | 11.4 KB       |
+java            | [Jackson](https://github.com/FasterXML/jackson-core)                             | 6373          | 13   KB       |
+C++             | [nlohmann::json](https://github.com/nlohmann/json)                               | 13787         | 27.6 KB       | segfault
+Nim             | [json](https://nim-lang.org/docs/json.html)                                      | 100000        | 200 KB        | w/ `-d:release`
+go              | `encoding/json`                                                                  | 2581101       | 5.0 MiB       | goroutine stack exceeds 1000000000-byte limit
+ruby            | [Oj](https://github.com/ohler55/oj)                                              | ∞             | ∞             |
+Haskell         | [Aeson](https://hackage.haskell.org/package/aeson)                               | ∞             | ∞             | available RAM is the only limit
+C#              | [System.Text.Json](https://docs.microsoft.com/en-us/dotnet/api/system.text.json) | ∞             | ∞             | with MaxDepth = int.MaxValue
```

Comment:
We are trying to compare the default options of each parser. Can you put the results of the parser without any options, and add a note about the possibility to set a `MaxDepth` ?